python/ml4ir/applications/ranking/tests/test_base.py

```diff
@@ -7,18 +7,30 @@
 import os
 import random
 from argparse import Namespace
-
-from ml4ir.base.io import file_io
+from tensorflow.keras.metrics import Metric
+from tensorflow.keras.optimizers import Optimizer
+
+from ml4ir.base.model.relevance_model import RelevanceModel
+from ml4ir.base.model.losses.loss_base import RelevanceLossBase
+from ml4ir.base.model.scoring.scoring_model import ScorerBase, RelevanceScorer
+from ml4ir.base.model.scoring.interaction_model import InteractionModel, UnivariateInteractionModel
+from ml4ir.base.model.optimizer import get_optimizer
+import ml4ir.base.io.file_io as file_io
 from ml4ir.base.io.logging_utils import setup_logging
-from ml4ir.base.config.parse_args import define_args
+from ml4ir.base.features.feature_config import FeatureConfig
+from ml4ir.applications.ranking.model.ranking_model import RankingModel
+from ml4ir.applications.ranking.model.losses import loss_factory
+from ml4ir.applications.ranking.model.metrics import metric_factory
+from ml4ir.applications.ranking.config.parse_args import get_args
 
 import warnings
+from typing import Union, List, Type
 
 warnings.filterwarnings("ignore")
 
 
-OUTPUT_DIR = "ml4ir/tests/test_output"
-ROOT_DATA_DIR = "ml4ir/tests/data"
+OUTPUT_DIR = "ml4ir/applications/ranking/tests/test_output"
+ROOT_DATA_DIR = "ml4ir/applications/ranking/tests/data"
 FEATURE_CONFIG_FNAME = "feature_config.yaml"
```

Comment:
should this be a static filename, or configurable as well?

Reply:
I don't understand why you would want to make the test data configurable? What are we testing in that case?

Reply:
nevermind, it's test, got it.